src/sentry/static/sentry/app/views/admin/adminWarnings.tsx

```diff
@@ -3,13 +3,16 @@ import React from 'react';
 import AsyncView from 'app/views/asyncView';
 import {t} from 'app/locale';
 
-export default class AdminSettings extends AsyncView {
-  getEndpoints() {
-    return [['data', this.getEndpoint()]];
-  }
+type Data = {
+  groups: [groupName: string, grouppedWarnings: string[]][];
+  warnings: string[];
+};
+
+type State = AsyncView['state'] & {data: Data};
 
-  getEndpoint() {
-    return '/internal/warnings/';
+export default class AdminSettings extends AsyncView<{}, State> {
+  getEndpoints(): [string, string][] {
```

Comment:
`ReturnType<AsyncView['getEndpoints']>` is the convention we've been use recently

Reply:
gonna batch fix these in a sec 👍